examples/crypto-perp-stream/crypto_stream.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alpacahq/alpaca-trade-api-go/v3/marketdata"
+	"github.com/alpacahq/alpaca-trade-api-go/v3/marketdata/stream"
+)
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	baseURL := "ws://stream.data.alpaca.markets/v1beta1/crypto-perps"
+
+	// Creating a client that connects to iex
```

Comment:
Not iex. But I think this comment is not useful anyway, so you can remove it completely.

Reply:
Done.